### PR TITLE
Add recommended labels check

### PR DESCRIPTION
**Pull request by anukul**
Related to https://github.com/keptn/keptn/issues/1958

### CODE REVIEW

File: pkg/kubernetes-utils_test.go [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+	"github.com/magiconair/properties/assert"
+	"helm.sh/helm/v3/pkg/chart/loader"
+)
+
+var recommendedLabelsCharts = []struct {
+	chartPath      string
+	expectedResult bool
+}{
+	{"testdata/all_recommended_labels", true},
+	{"testdata/some_recommended_labels", false},
```
Comment (anukul) [inline pkg/kubernetes-utils_test.go:15]: I used `helm create` to generate charts. These charts had all but 2 of the recommended labels (`app.kubernetes.io/component`, `app.kubernetes.io/part-of`) which I added by myself.

File: pkg/testdata/all_recommended_labels/templates/_helpers.tpl [hunk 2]
```diff
@@ -0,0 +1,64 @@
...
+Common labels
+*/}}
+{{- define "all_recommended_labels.labels" -}}
+helm.sh/chart: {{ include "all_recommended_labels.chart" . }}
+{{ include "all_recommended_labels.selectorLabels" . }}
+{{- if .Chart.AppVersion }}
+app.kubernetes.io/version: {{ .Chart.AppVersion | quote }}
+{{- end }}
+app.kubernetes.io/component: some-component
+app.kubernetes.io/part-of: kubernetes-utils-tests
```
Comment (anukul) [inline pkg/testdata/all_recommended_labels/templates/_helpers.tpl:43]: See https://github.com/keptn/kubernetes-utils/pull/8/files#r497186796

**Comment by anukul** [comment]
cc @christian-kreuzberger-dtx

**Comment by anukul** [comment]
Converted to draft because this might change depending on the answer to https://github.com/keptn/keptn/issues/1958#issuecomment-701649780

**Comment by christian-kreuzberger-dtx** [comment]
Hi @anukul , are you still planning to update this PR?
IMHO we can merge it as it is, and work on it later.

**Review by kadirselcuk — APPROVED** [review]

**Pull request closed without merging**